modules/deployment-scripts/wait/README.md

```diff
@@ -26,7 +27,7 @@ When deploying Azure services together, it is sometimes necessary to introduce d
 param location string = resourceGroup().location
 param waitSeconds int =  60
 
-module delayDeployment 'br/public:deployment-scripts/wait:1.0.1' = {
+module delayDeployment 'br/public:deployment-scripts/wait:1.0.2' = {
```

Comment:
I think we should bump the version to `1.1` since a new parameter with a default value is added.

Reply:
Done @shenglol